.github/scripts/publish_github_action.sh

```diff
@@ -26,34 +26,34 @@ else
     npm publish
 fi
 
-# TODO loop through checking for the status instead of sleeping
-echo -e "sleeping for 30 seconds to ensure azle@$VERSION is fully registered on npm"
+# # TODO loop through checking for the status instead of sleeping
```

Comment:
Should we delete this comment now? Why keep it?

Reply:
I'll do you one better, we can delete the whole file

Reply:
Please actually delete

Reply:
done